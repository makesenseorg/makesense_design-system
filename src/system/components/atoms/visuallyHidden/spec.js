import { shallowMount } from "@vue/test-utils";
import Comp from "./VisuallyHidden.vue";

describe("VisuallyHidden.vue", () => {
  it("matches snapshot", () => {
    const wrapper = shallowMount(Comp, {
      slots: {
        default: "Lorem ipsum dolor sit amet",
      },
    });
    expect(wrapper.element).toMatchSnapshot();
  });

  it("display content in slot", () => {
    const wrapper = shallowMount(Comp, {
      slots: {
        default: "Lorem ipsum dolor sit amet",
      },
    });
    expect(wrapper.text()).toBe("Lorem ipsum dolor sit amet");
  });
});
