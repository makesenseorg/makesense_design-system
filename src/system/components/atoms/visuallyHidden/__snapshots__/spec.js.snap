// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VisuallyHidden.vue matches snapshot 1`] = `
<div
  class="visually-hidden"
>
  Lorem ipsum dolor sit amet
</div>
`;
